R/desc.R

```diff
@@ -133,25 +133,47 @@ process_formatting.desc_layer <- function(x, ...) {
       # trans_sums[[i]] <- trans_sums[[i]] %>%
       #   mutate(display_string = str_pad(display_string, max_layer_length, side='right'))
 
+
+
+
       # Now do one more transpose to split the columns out
       # Default is to use the treatment variable, but if `cols` was provided
       # then also transpose by cols.
-      form_sums[[i]] <- trans_sums[[i]] %>%
-        pivot_wider(id_cols=c('row_label', match_exact(by)), # Keep row_label and the by variables
-                    names_from = match_exact(vars(!!treat_var, !!!cols)), # Pull the names from treatment and cols argument
-                    names_prefix = paste0('var', i, "_"), # Prefix with the name of the target variable
-                    values_from = display_string # Use the created display_string variable for values
+      if (('stats_as_columns' %in% ls()) && stats_as_columns) {
```

Comment:
```suggestion
      if ((stats_as_columns) {
```
I'm thinking we can set `stats_as_columns <- FALSE` in zzz.R and then here we can remove the call to `ls`

Reply:
Good call - fixed